src/pages/clouddocs/troubleshooting.md

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
There's a monitoring and troubleshooting section. Wondering if this should be merged under that somehow? Maybe best to get content moved over and we can improve the flow/presentation of pages afterwards.

Reply:
I believe the clouddocs troubleshooting page can be removed however if we want to keep the content then ;yes it should be merged into the container edition troubleshooting section.